sirepo/package_data/static/json/opal-schema.json

```diff
@@ -291,9 +291,8 @@
         ],
         "ParticlematterinteractionType": [
             ["", "NOT SELECTED"],
-            ["CCOLLIMATOR", "CCOLLIMATOR"],
```

Comment:
I believe there should be a fixup made because these types have been removed. Idk if you've written a fixup before. If not I'm happy to share more info on how they're done.

Reply:
I'll reach out to you when I fix the test failures. (maybe related to fixup) Thanks @e-carlin

Reply:
I removed DEGRADER files from the import test, if all tests pass, then I'll reach out about the fixup. In addition to removing references to the old particlematterinteractionTypes, will it be necessary to write tests involving SCATTERING and BEAMSTRIPPING types?

Reply:
I don't have the best intuition for what will break so what should be tested. But, I think adding a test for one of SCATTERING OR BEAMSTRIPPING would be good. Some thoughts for writing the tests: change all instances of DEGRADER in the old test data to one of the new words. My guess is things will work since we are just parsing and not running opal. If we were running it may produce something invalid. Another idea is to go to the opal docs. Many codes have examples with their docs.